dataactvalidator/config/sqlrules/a16_appropriations.sql

```diff
@@ -2,12 +2,15 @@
 WITH appropriation_a16_{0} AS
     (SELECT submission_id,
         row_number,
-        budget_authority_unobligat_fyb
+        budget_authority_unobligat_fyb,
+        tas,
```

Comment:
Why are we pulling `tas` in here? We don't use it

Reply:
Most of the time the TAS was used to join as well. Good catch.